### PR TITLE
Modernize Python syntax, add Python 3.8

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
-sudo: True
+version: ~> 1.0
 dist: xenial
```
Comment (auvipy) [inline .travis.yml]: make it bionic

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+os: linux
 language: python
 cache: pip
 python:
-- 3.7
 - 3.6
+- 3.7
+- 3.8-dev
```
Comment (auvipy) [inline .travis.yml]: this should 3.8 only

File: .travis.yml (hunk 1 above)
Comment (ad-m) [reply .travis.yml]: Done.

File: .travis.yml (hunk 2 above)
Comment (ad-m) [reply .travis.yml]: Done. I was not aware that TravisCI added it already.

**Comment by auvipy** [comment]
all the travis builds are failing

**Comment by ad-m** [comment]
@auvipy : fixed.

**Pull request merged**